tools/unit_test_runner.sh

```diff
@@ -84,7 +84,7 @@ for pkg in $flaky_tests; do
   max_attempts=3
   attempt=1
   # Set a timeout because some tests may deadlock when they flake.
-  until go test -timeout 2m $VT_GO_PARALLEL $pkg -v -count=1; do
+  until go test -timeout 5m $VT_GO_PARALLEL $pkg -v -count=1; do
```

Comment:
The new unit test takes longer than the average one, and with this new test we would often go over the 2m mark.